ui/app/AppLayouts/Communities/CommunitiesPortalLayout.qml

```diff
@@ -107,49 +112,45 @@ StatusSectionLayout {
                 color: Theme.palette.directColor1
             }
 
-            RowLayout {
-                Layout.fillWidth: true
-                Layout.preferredHeight: 38
-                spacing: Theme.bigPadding
+            ColumnLayout {
+                spacing: Theme.padding
+
+                RowLayout {
+                    LayoutItemProxy {
```

Comment:
Probably we don't need this proxy, right? I see it has a constant target

Reply:
Done!